app/src/main/java/com/example/lookup/view/closet/MyCloth.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.lookup.view.closet
+
+import java.io.Serializable
+
+data class MyCloth(var img: Int, var name:String): Serializable
```

Comment:
```suggestion
data class MyCloth(var img: Int, var name:String, var filename:String): Serializable
```
이렇게 MyCloth에 파일이름이 추가되어서 클릭했을 때, sharedPreference에 값 설정할 수 있어야 된다는 의미입니다! 현재는 서버에서 어떤 파일을 가져와야된다는 정보가 없어서 클릭 시에 아무런 동작을 할 수 가 없어요!
즉
```
PreferenceManager.setString(requireContext(),CLOTH_FILENAME,"white_sneakers.obj")
```
여기서 "white_sneakers.obj"대신에 들어갈 값도 MyCloth에서 가지고 있어야 된다는 의미입니다. 나중에 이부분을 data.filename으로 바꿔야 되고요